en/peripherals/frsky_telemetry.md

```diff
@@ -1,27 +1,65 @@
 # FrSky Telemetry
 
-PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards
-(Pixracer) include the required electronics for the interface, older
-boards (Pixhawk) need a UART to S.PORT adapter board.
+FrSky telemetry allows you to access vehicle telemetry and error information on a compatible RC transmitter.
 
+This information includes: flight mode, battery level, RC signal strength, speed, altitude etc. Some transmitters can additionally provide audible and vibration feedback, which is particularly useful for low battery and other failsafe warnings.
 
-## Pixracer
+## Flight Controller Hardware
 
-> **Success** FrSky telemetry auto-starts on Pixracer and
-  detects D or S mode, there is no need to configure anything.
+Pixhawk/PX4 supports D (old) and S (new) FrSky telemetry. FMUv4 based boards (Pixracer) include the required electronics for the interface, older boards (Pixhawk) need a UART to S.PORT adapter board.

```

Comment:
What else does S offer PX4 users - or is it irrelevant other than for connections?

Reply:
OK. Doesn't seem all that useful to us since we can already send out this info (without having to add sensors to a bus).